PersonListener/Domain/TenureInformation/TenureType.cs

```diff
@@ -0,0 +1,9 @@
+namespace PersonListener.Domain.TenureInformation
+{
+    public class TenureType
+    {
+        public string Code { get; set; }
+
+        public string Description { get; set; }
```

Comment:
Do we need to add the TenureTypes logic to calculate PersonTenureType?

Reply:
No. The `PersonTenureType` is on the Tenure `HouseholdMembers` data. We are retrieving this data and not setting it.
The `PersonTenureType` value we retrieve will be a value within the `PersonType` enum.